src/modules/graphics/wrap_Graphics.cpp

```diff
@@ -3255,6 +3283,16 @@ int w_polygon(lua_State *L)
 	return 0;
 }
 
+int w_dispatchThreadgroups(lua_State* L)
+{
+	Shader *shader = luax_checkshader(L, 1);
+	int x = luaL_optinteger(L, 2, 1);
```

Comment:
I think it makes sense to require threadgroupsX and keep y/z optional

Reply:
Also `optinteger` returns a 64 bit integer (`lua_Integer`) on 64 bit platforms, so there should be an explicit cast down to `int` here.